app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/autofill/AutoFillScreen.kt

```diff
@@ -79,6 +82,12 @@ fun AutoFillScreen(
             AutoFillEvent.NavigateToBlockAutoFill -> {
                 onNavigateToBlockAutoFillScreen()
             }
+
+            AutoFillEvent.NavigateToSettings -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+                    CredentialManager.create(context).createSettingsPendingIntent().send()
```

Comment:
Most intent creation is done in `IntentManager`. We can also perform the build version check there, if we need it.

Reply:
Ok, sounds good! I didn't know if this method belonged there since `CredentialManager` is slightly different than a regular intent, but I like that better so I'll move it to the `IntentManager`!